lua/damagelogs/server/autoslay.lua

```diff
@@ -530,7 +531,7 @@ end)
 
 if Damagelog.ULX_Autoslay_ForceRole then
     hook.Add("Initialize", "Autoslay_ForceRole", function()
-        if not ROLES then
+        if not TTT2 and not CR_VERSION then
```

Comment:
Does this fix the issue you mentioned before?
> Solves #55 as long as `Damagelog.ULX_Autoslay_ForceRole` is set to false

Reply:
It does. Apparently it already skipped the check with TTT2 using the old constant. I just updated it to use the new constant and check for CR